src/utils.ts

```diff
@@ -64,12 +64,28 @@ const sanitizeMessage = (message: string) => message.trim().replace(/[\n\r]/g, '
 
 const promptTemplate = 'Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything:';
```

Comment:
Convert this to an arrow function that accepts the language and the diff

Reply:
Done!